flypipe/node.py

```diff
@@ -12,6 +14,9 @@
 from flypipe.utils import DataFrameType
 
 
+logger = logging.getLogger(__name__)

```

Comment:
What is the motivation on start using logger?

Reply:
Just because I noticed it's a pain at the moment to understand what Flypipe is doing when running it somewhere without breakpoints like Databricks. Logging is good as we can get a trail of what Flypipe is doing and this is visible in code that is running concurrently. Also logging levels are handy, by default info logs like these won't show up but if we set the log level to debug or info then we get all this info in the Databricks command execution.